<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qmain_email</class>
 <widget class="QWidget" name="Qmain_email">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>qtCRMInterface</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>961</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Qhoriz_layout_main" stretch="1,5">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="Qvert_layout_actions" stretch="1,1,1,8,1,2">
      <item>
       <widget class="QPushButton" name="Qbutton_save">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbutton_abort">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Qlabel_obs">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Observatii:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="Qtext_obs"/>
      </item>
      <item>
       <widget class="QLabel" name="Qlabel_error">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Erori:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="Qtext_error"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Qvert_layout_boxes" stretch="1,1,1,1,1,1,1,1,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_name_p">
        <item>
         <widget class="QLabel" name="Qlabel_name_p">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nume persoana:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_name_p">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_dates">
        <item>
         <widget class="QLabel" name="Qlabel_dates">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Date persoana:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_dates">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_name_f">
        <item>
         <widget class="QLabel" name="Qlabel_name_f">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nume firma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_name_f">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_cui">
        <item>
         <widget class="QLabel" name="Qlabel_cui">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CUI firma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_cui">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_email">
        <item>
         <widget class="QLabel" name="Qlabel_email">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>E-mail firma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_email">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_tel">
        <item>
         <widget class="QLabel" name="Qlabel_tel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Telefon firma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_tel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_num_a">
        <item>
         <widget class="QLabel" name="Qlabel_num_a">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Numar angajati:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_num_a">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_caen">
        <item>
         <widget class="QLabel" name="Qlabel_caen">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Cod CAEN:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_caen">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Qlayout_fund">
        <item>
         <widget class="QLabel" name="Qlabel_fund">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ce se doreste a fi finantat:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="Qtext_fund">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
